flight_backend/api/serializers/users.py

```diff
@@ -39,6 +39,18 @@ def create(self, validated_data):
         user.save()
         return user
 
+    def update(self, instance, validated_data):
```

Comment:
any reason we're overriding the default update method?

Reply:
I think you have to provide logic for the update method. I am not sure it works if you dont define it.

Reply:
you can check this out https://www.django-rest-framework.org/api-guide/serializers/

Reply:
sorry.. just saw you're using `APIView` on for the viewset. will you need to allow users to be added using means other than registration?

Reply:
No, Users will be added by registration alone.